### ISSUE TITLE
Config file 

### ISSUE THREAD

**Issue by deadbits** [needmorecowbell/sniff-paste#5]
It'd be great if the regex patterns were moved to a config file, and users could also specify direct keywords they want to find, so you don't need to edit the source code to make changes, remove patterns, etc.

You already covered this in another issue I think but the addition of YARA signature scanning of pastes would be fantastic as well. Specify rules path in config, along with the stuff above.

**Comment by needmorecowbell** [comment]
Good call, gonna make this a higher priority issue. It'll go hand in hand with your pro version issue as well. That one might take more time
